service/src/java/org/apache/hive/service/cli/HiveSQLException.java

```diff
@@ -150,4 +191,13 @@ public static TStatus toTStatus(Exception e) {
     return tStatus;
   }
 
+  @Override
+  public String getMessage(){
+    String errorMsg = super.getMessage() == null ? "Error" : super.getMessage();
+    if (!errorMsg.contains(ServiceContext.QUERY_ID)) {
+      return String.format("%s; Query ID: %s", errorMsg, queryId);
```

Comment:
Shouldn't we check ``queryId`` not null as well? else someone having an old jar which creates this HiveSqlException without QueryId will start getting this with null?